### PR TITLE
[FEATURE] restyle approvalDetail

### PR DESCRIPTION
**Pull request by sarykemmoun**
I’ve restyled the approval details component.
I’ve changed the comment from modal to input, and I’ve increased the buttons.
Therefore I’ve created another approvals button in approval details, to simplify the code.

I’ve restyled also the requesterRecipient component

attached  approval details in 2 different modes:
the first one is pending  approval and the second - approval
screenshot_1509031330
screenshot_1509030503

### CODE REVIEW

File: js/components/ApprovalsDetails/Banner.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ import { Icon } from 'react-native-elements';
 import PropTypes from 'prop-types';
 import { white } from '../../utils/colors';
 
-function Banner({ backgroundColor, iconName, iconType, text }) {
+function Banner({ backgroundColor, iconName, iconType, text, response }) {
     const { appointment, flexRow, iconStyle, textStyle } = styles;
 
     return (
-        <View style={[appointment, { backgroundColor }]}>
+        response !== null && <View style={[appointment, { backgroundColor }]}>
```
Comment (raphaelboukara) [inline js/components/ApprovalsDetails/Banner.js]: ??? did u test it?

File: js/components/ApprovalsDetails/BannerContainer.js [hunk 2]
```diff
@@ -28,7 +28,8 @@ const mapStateToProps = ({ approvals }, ownProps) => {
         text,
         backgroundColor,
         iconName,
-        iconType
+        iconType,
+        response
```
Comment (raphaelboukara) [inline js/components/ApprovalsDetails/BannerContainer.js]: what's the pblem with response? what the possible values for response?? 
BTW: `if(!!response) <=> if(response === true)`

**Comment by raphaelboukara** [comment]
please don't forget to rebase before pr

File: js/components/ApprovalsDetails/Banner.js (hunk 1 above)
Comment (sarykemmoun) [reply js/components/ApprovalsDetails/Banner.js]: Yes. added a screenshot 
screenshot_1509259920

**Pull request merged**